rero_ils/modules/stats/views.py

```diff
@@ -80,6 +83,63 @@ def stats_librarian():
         type='librarian')
 
 
+@blueprint.route('/librarian/<record_pid>/csv')
+@check_logged_as_librarian
+def stats_librarian_queries(record_pid):
+    """Download specific statistic query into csv file.
+
+    :param record_pid: statistics pid
+    :return: response object, the csv file
+    """
+    queries = {'1': 'loans_of_transaction_library_by_item_location'}
+    query_id = request.args.get('query_id', None)
```

Comment:
maybe add an USAGE section into the function doc_string to explain `query_id` query string argument.